jenkins-slaves/jenkins-slave-npm/Dockerfile

```diff
@@ -23,4 +20,8 @@ RUN yum install -y centos-release-scl-rh && \
     yum clean all -y && \
     rm -rf /var/cache/yum
 
-USER 1001
+USER root
+RUN npm install --unsafe-perm -g npm-audit-html npm-audit-ci-wrapper sonar-scanner || cat /home/jenkins/.npm/_logs/*-debug.log
+RUN sonar-scanner || echo
```

Comment:
Please consolidate all `RUN` statements into one, similar to `Dockerfile.rhel7`.
Also no need for `USER root` here.